addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -34,6 +34,12 @@ export class MassMailingHtmlField extends HtmlField {
         this.orm = useService('orm');
         this.dialog = useService('dialog');
 
+        this.onIframeUpdated = this._updateIframe.bind(this);
```

Comment:
I guess to be "correct" we might want to call the original function too, if possible